Sources/IO/Core/DataAccessHelper/JSZipDataAccessHelper.js

```diff
@@ -72,7 +72,9 @@ const handlers = {
 };
 
 function removeLeadingSlash(str) {
-  return str[0] === '/' ? str.substr(1) : str;
+  // Hacky way to normalize the path. TODO: ask about this.
+  const url = new URL(str, 'http://any').pathname;
+  return url[0] === '/' ? url.substr(1) : url;
```

Comment:
@floryst @jourdain any suggestion ?

Reply:
To expand, this is needed because we'd like to be able to use relative paths (..) in .vtkjs files, but jszip does not seem to support them.

Reply:
If you are just removing slashes from the beginning of a string, try `str.replace(/^\//, '')`.

Reply:
I would rather keep it that way then, if that's ok with you